### ISSUE TITLE
add python dependencies

### ISSUE THREAD

**Issue by dwks** [daanzu/kaldi-active-grammar#27]
I just installed kaldi-active-grammar on a new system. The prebuilt pip package didn't depend on `sounddevice` and `webrtcvad`, and I got runtime errors from my grammar until I installed them. It would be good to list these as python dependencies in the pip package, assuming they would be required by any grammar.

**Comment by daanzu** [comment]
kaldi-active-grammar doesn't depend on those, since it is supposed to be agnostic as to the source. If using dragonfly, I think they should be installed with `pip install dragonfly2[kaldi]`.

**Comment by dwks** [comment]
Yes, this works. Please consider adding this to the README, it's pretty confusing to start using kaldi-active-grammar on Linux currently. P.S. `pip install dragonfly2[kaldi]` pulls in `kaldi-active-grammar` automatically.

**Comment by dwks** [comment]
If you don't want to mention dragonfly on the kaldi-active-grammar page, could at least mention it here: https://github.com/daanzu/kaldi-grammar-simple

**Comment by daanzu** [comment]
I should definitely make this more clear. Thanks for pointing it out!
